projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -162,10 +165,26 @@ class LoggingPlugin:
 
     @hookimpl
     def vdk_configure(self, config_builder: ConfigurationBuilder):
-        config_builder.add(key=SYSLOG_URL, default_value="localhost")
-        config_builder.add(key=SYSLOG_PORT, default_value=514)
-        config_builder.add(key=SYSLOG_ENABLED, default_value=False)
-        config_builder.add(key=SYSLOG_SOCK_TYPE, default_value="UDP")
+        config_builder.add(
+            key=SYSLOG_URL,
+            default_value="localhost",
+            description="The URL of the endpoint to which VDK logs will be sent through SysLog.",
```

Comment:
Reading the documentaiton, we say url, but it's really just host name , right .

When you say URL people except something like syslog://abv.bg/xx or similar ?